### PR TITLE
Feature/menu bar

### PR DESCRIPTION
**Pull request by sanaf1333**

### CODE REVIEW

**Comment by evshi** [comment]
@sanaf1333 when merging this branch I got a ton of eslint errors

File: packages/runtime/src/widgets/DataGrid.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
+import React from "react";
```
Comment (evshi) [inline packages/runtime/src/widgets/DataGrid.tsx:1]: Please remove this file

**Comment by evshi** [comment]
A few small things I noticed during testing:

- Side menu height is not responsive
- Need support to specify different icons
- Collapse animation is a little janky because the text sizing isn't smooth

Also, let's move this up to the View widget property like we discussed and render this from the EnsembleScreen as part of the frame.

**Comment by sanaf1333** [comment]
> @sanaf1333 when merging this branch I got a ton of eslint errors

can you tell me what eslint errors you are getting?

**Comment by evshi** [comment]
Closing since this was merged with #35

**Pull request closed without merging**